csrc/scheduler/normalization_utils.cpp

```diff
@@ -1630,13 +1621,30 @@ void schedulePersistentKernel(
       reduction_scheduler_utils::getCachedTvsToUnrollOrVectorize(
           reference_tv, is_vectorize, cached_inputs, cached_outputs);
 
+  std::unordered_set<TensorView*> unroll_vectorizable_cached_and_cast_tvs;
```

Comment:
Should we insert `unroll_vectorizable_cached_tvs` into `unroll_vectorizable_cached_and_cast_tvs`?